easybuild/easyconfigs/h/hevea/hevea-2.36-GCC-13.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'hevea'
+version = '2.36'
+
+homepage = 'http://pauillac.inria.fr/~maranget/hevea/'
+description = """A quite complete and fast LATEX to HTML translator"""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+source_urls = ['http://pauillac.inria.fr/~maranget/hevea/distri']
+sources = [SOURCE_TAR_GZ]
+checksums = ['5d6759d7702a295c76a12c1b2a1a16754ab0ec1ffed73fc9d0b138b41e720648']
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('ocamlbuild', '0.14.3'),
+]
+
+osdependencies = ['texlive']
```

Comment:
@PetrKralCZ can this use the easyconfig dependency instead of osdep?

Reply:
texlive easyconfig being contributed in #20406 should work

Reply:
```suggestion
dependencies = [
    ('texlive', '20230313'),
]